examples/test-fieldgroup/runtest

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+#
+# This scipt is intended to be quite quiet when it succeeds,
+# printing only the test name and "Passed". 
+# If the test fails then the whole log is printed, hopefully
+# helping diagnose where the problem lies.
+#
+
+echo ""
+echo "Running FieldGroup test"
+
+# Ensure we clean old run, in case it succeeded last time
+make -f makefile_fail clean
+
+# This test should fail to compile
+make -f makefile_fail > run.log  2>&1
+
+if [ $? -eq 0 ]; then
+  # Compiled ok, something wrong
+  # Send the run.log file to stdout so it's stored in Travis logs
+  echo "Fail. Log follows:"
+  echo ""
+  cat run.log
+  exit 1;
+fi
+
+# Second test should pass
+
+make clean
+make >> run.log 2>&1
```

Comment:
Not sure why the forced rebuild is needed
I have disabled the build in test_suite_build to prevent it from being build twice